<?xml version="1.0"?>
<launch>
  <!-- <master auto="start"/>  -->
<!--   <param name="use_sim_time" value="true" /> -->

  <!--   map_server -->
  <node pkg="map_server" type="map_server" name="map_server" args="$(find patrol_robot)/map/mapv0.2.yaml"/>

  <!--   amcl -->
  <node pkg="amcl" type="amcl" name="amcl" output="screen">
    <param name="initial_pose_x" value="0" />
    <param name="initial_pose_y" value="0" />
    <param name="initial_pose_a" value="0" />
    <param name="initial_pose_xx" value="0.01" />
    <param name="initial_pose_yy" value="0.01" />
    <param name="initial_pose_aa" value="0.01" />
  </node>

  <!--   move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find patrol_robot)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find patrol_robot)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find patrol_robot)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find patrol_robot)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find patrol_robot)/param/base_local_planner_params.yaml" command="load" />
    <param name="controller_frequency" value="5" />
    <param name="controller_patience" value="15" />
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS" />

  </node>

  <node pkg="patrol_robot" type="fixed_traj_nav.py" name="fixed_traj_nav" respawn="false" output="screen" />

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find patrol_robot)/rviz_cfg/fixed_traj.rviz"/>
</launch> 

